x/evm/keeper/funtoken_from_coin_test.go

```diff
@@ -304,60 +305,86 @@ func (s *FunTokenFromCoinSuite) TestNativeSendThenPrecompileSend() {
 		deps.App.BankKeeper,
 		deps.Ctx,
 		testContractNibiAddr,
-		sdk.NewCoins(sdk.NewCoin(bankDenom, sdk.NewInt(10e6)))),
+		sdk.NewCoins(sdk.NewCoin(bankDenom, sdk.NewIntFromBigInt(sendAmt)))),
+	)
+	evmtest.AssertBankBalanceEqual(
+		s.T(), deps, bankDenom, testContractAddr, sendAmt,
+	)
+	evmtest.AssertBankBalanceEqual(
+		s.T(), deps, bankDenom, evm.EVM_MODULE_ADDRESS, big.NewInt(0),
 	)
 
 	s.T().Log("Convert bank coin to erc-20: give test contract 10 WNIBI (erc20)")
 	_, err = deps.EvmKeeper.ConvertCoinToEvm(
 		sdk.WrapSDKContext(deps.Ctx),
 		&evm.MsgConvertCoinToEvm{
 			Sender:    deps.Sender.NibiruAddr.String(),
-			BankCoin:  sdk.NewCoin(bankDenom, sdk.NewInt(10e6)),
+			BankCoin:  sdk.NewCoin(bankDenom, sdk.NewIntFromBigInt(sendAmt)),
 			ToEthAddr: eth.EIP55Addr{Address: testContractAddr},
 		},
 	)
 	s.Require().NoError(err)
+	evmtest.FunTokenBalanceAssert{
+		FunToken:     funtoken,
+		Account:      testContractAddr,
+		BalanceBank:  sendAmt,
+		BalanceERC20: sendAmt,
+	}.Assert(s.T(), deps)
+	evmtest.FunTokenBalanceAssert{
+		FunToken:     funtoken,
+		Account:      evm.EVM_MODULE_ADDRESS,
+		BalanceBank:  sendAmt,
+		BalanceERC20: big.NewInt(0),
+	}.Assert(s.T(), deps)
 
 	// Alice hex and Alice bech32 is the same address in different representation,
 	// so funds are expected to be available in Alice's bank wallet
 	alice := evmtest.NewEthPrivAcc()
+	evmtest.FunTokenBalanceAssert{
+		FunToken:     funtoken,
+		Account:      alice.EthAddr,
+		BalanceBank:  big.NewInt(0),
+		BalanceERC20: big.NewInt(0),
+	}.Assert(s.T(), deps)
```

Comment:
Much easier to read IMO

Reply:
One improvement could be to add a field to `FunTokenBalanceAssert ` for a custom description message, such as `description: "Alice should have zero NIBI"`. It would make reading through a series of struct assertions much quicker imo.

Reply:
Yeah, I like that. And since it's a struct, the field is actually optional. Adding now